modules/expose-portal-http-during-installation.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * dir/filename.adoc
+:_module-type: PROCEDURE
+[id="expose-portal-http-during-installation_{context}"]
+= Exposing RHACS portal over HTTP during the installation
+
+[role="_abstract"]
+If you are installing {product-title} using the `roxctl` CLI, to enable the HTTP server during deployment, use the `--plaintext-endpoints` option with the `roxctl central generate interactive` command.
```

Comment:
Saying, "If you are" then, "to enable" doesn't sound right to me. I'd either try to reword like how you did the proxy one, or reorder the second and third parts of the sentence:

```
If you are installing {product-title} using the `roxctl` CLI, use the `--plaintext-endpoints` option with the `roxctl central generate interactive` command to enable the HTTP server during deployment.
```

I would also suggest "installation" instead of "deployment"/